src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+        _drive           = drive;
+        _ShooterBed      = shooterBed;
+        _notepath        = notepath;
+        _ShooterFlywheel = ShooterFlywheel;
+        _Intake          = intake;
+        _climb           = climb;
+
+        var tab         = Shuffleboard.getTab("Dashboard");
+        var SettingsTab = Shuffleboard.getTab("Settings");
+
+        // Camera Stream
+        tab.addCamera("Camera Stream", Constants.Vision.CAMERA_NAME, "mjpg:http://10.5.25.12:1181/?action=stream").withWidget("Camera Stream").withPosition(0, 0).withSize(15, 10)
+                .withProperties(Map.of("Show crosshair", false, "Show controls", false));
+        // Field
+        _field = new Field2d();
+        tab.add("field", _field).withPosition(15, 0).withSize(15, 8);
+
+        // Alliance, Note, and Intake boxes
+        var booleanBoxLayout = tab.getLayout("Color Box", BuiltInLayouts.kGrid).withPosition(15, 12).withSize(5, 3).withProperties(Map.of("Number of columns", 1, "Number of rows", 2, "Label position", "LEFT"));
+        _allianceBox = booleanBoxLayout.add("Alliance", false).withPosition(0, 0).withWidget(BuiltInWidgets.kBooleanBox).withProperties(Map.of("Color when true", "blue", "Color when false", "red")).getEntry();
+        _HasNote     = booleanBoxLayout.add("Has Note", false).withPosition(0, 1).withWidget(BuiltInWidgets.kBooleanBox).withProperties(Map.of("Color when true", "green", "Color when false", "red")).getEntry();
+
+        // Swerve Module Angles
+        var driveSettingsLayout = tab.getLayout("Drive Subsystem", BuiltInLayouts.kGrid).withPosition(15, 8).withSize(5, 4);
+        _flAngle = driveSettingsLayout.add("FL Angle", 0.0).withPosition(0, 0).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _frAngle = driveSettingsLayout.add("FR Angle", 0.0).withPosition(2, 0).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _blAngle = driveSettingsLayout.add("BL Angle", 0.0).withPosition(0, 2).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _brAngle = driveSettingsLayout.add("BR Angle", 0.0).withPosition(2, 2).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        // Height, Intake and Speed
+        var heightAndIntakeLayout = tab.getLayout("Height and Intake", BuiltInLayouts.kGrid).withPosition(0, 10).withSize(15, 5).withProperties(Map.of("Number of columns", 3, "Number of rows", 1));
+        _LeftHeight  = heightAndIntakeLayout.add("Left Height", 0).withSize(5, 4).withPosition(0, 0).withWidget(BuiltInWidgets.kNumberBar).withProperties(Map.of("Orientation", "VERTICAL", "Min", 0, "Max", 25)).getEntry();
+        _Rightheight = heightAndIntakeLayout.add("Right Height", 0).withWidget(BuiltInWidgets.kNumberBar).withPosition(2, 0).withProperties(Map.of("Orientation", "VERTICAL", "Min", 0, "Max", 25)).getEntry();
+        _IntakeSpeed = heightAndIntakeLayout.add("Intake Speed", 0).withPosition(1, 0).withWidget(BuiltInWidgets.kNumberBar).getEntry();
```

Comment:
A few things:
1. Updated tab name
2. Updated variable names
3. Updated displayed names
4. `withSize` not needed inside a layout
5. Lets be explicit with the ranges on number bars
6. Lets add the widgets in the order they'll appear in the layout

```suggestion
        // Height, Intake and Speed
        var heightAndIntakeLayout = dashboard.getLayout("Climb and Intake", BuiltInLayouts.kGrid).withPosition(0, 10).withSize(15, 5).withProperties(Map.of("Number of columns", 3, "Number of rows", 1));
        _leftHeight  = heightAndIntakeLayout.add("Left Height", 0).withPosition(0, 0).withWidget(BuiltInWidgets.kNumberBar).withProperties(Map.of("Orientation", "VERTICAL", "Min", 0, "Max", 25)).getEntry();
        _intakeSpeed = heightAndIntakeLayout.add("Intake Speed", 0).withPosition(1, 0).withWidget(BuiltInWidgets.kNumberBar).withProperties(Map.of("Min", -1, "Max", 1)).getEntry();
        _rightheight = heightAndIntakeLayout.add("Right Height", 0).withPosition(2, 0).withWidget(BuiltInWidgets.kNumberBar).withProperties(Map.of("Orientation", "VERTICAL", "Min", 0, "Max", 25)).getEntry();
```